### PR TITLE
[5491] Text consistency in the UI

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/5491

### CODE REVIEW

File: ui/app/src/components/ItemDisplay/utils.tsx [hunk 1]
```diff
@@ -48,13 +48,13 @@ export function getItemStateText(stateMap: ItemStateMap, values?: Record<string,
       stateMap.submitted ? (
         <FormattedMessage id="itemState.submittedToStaging" defaultMessage="Submitted to staging" />
       ) : (
-        <FormattedMessage id="itemState.scheduledForStaging" defaultMessage="Scheduled for staging" />
+        <FormattedMessage id="itemState.scheduledToStaging" defaultMessage="Scheduled to staging" />
```
Comment (rart) [inline ui/app/src/components/ItemDisplay/utils.tsx:51]: You must update this "key" on all localization json files. All changes in keys or defaultMessage must be replicated to the json files if they are there already.

File: ui/app/src/translations/locales/es.json [hunk 2]
```diff
@@ -525,7 +525,7 @@
   "itemState.scheduledForStaging": "Programado para puesta en escena",
   "itemState.scheduledToGoLive": "Programado para vivir",
   "itemState.submitted": "Presentado",
-  "itemState.submittedToLive": "Enviado para vivir",
+  "itemState.submittedToLive": "Enviado a vivir",
```
Comment (rart) [inline ui/app/src/translations/locales/es.json]: Update the translation to "Enviado a producción" or some better translation for live

File: ui/app/src/components/ItemStatesManagement/ItemStatesManagement.tsx [hunk 3]
```diff
@@ -268,7 +268,7 @@ export default function ItemStatesManagement(props: ItemStatesManagementProps) {
   return (
     <section ref={rootRef} className={classes.root}>
       <GlobalAppToolbar
-        title={!embedded && <FormattedMessage id="siteTools.itemStates" defaultMessage="Item States" />}
+        title={!embedded && <FormattedMessage id="siteTools.workflowStates" defaultMessage="Workflow States" />}
```
Comment (rart) [inline ui/app/src/components/ItemStatesManagement/ItemStatesManagement.tsx]: Check ui.xml of sites to see if this is somewhere around there too or in the defineMessages for ui config.

**Review by rart — CHANGES_REQUESTED** [review]

File: ui/app/src/components/ItemStatesManagement/ItemStatesManagement.tsx (hunk 3 above)
Comment (jvega190) [reply ui/app/src/components/ItemStatesManagement/ItemStatesManagement.tsx]: Not found neither in ui.xml nor in studio-ui. 
In ui.xml the message id is `workflowStates.title` (that's in defineMessages for ui config and in localization json files)

Comment (rart) [reply ui/app/src/components/ItemStatesManagement/ItemStatesManagement.tsx]: Let's use the same id here or there; no need for duplicate strings with the same text and target

File: ui/app/src/components/ItemStatesManagement/ItemStatesManagement.tsx [hunk 4]
```diff
@@ -268,7 +268,7 @@ export default function ItemStatesManagement(props: ItemStatesManagementProps) {
   return (
     <section ref={rootRef} className={classes.root}>
       <GlobalAppToolbar
-        title={!embedded && <FormattedMessage id="siteTools.itemStates" defaultMessage="Item States" />}
+        title={!embedded && <FormattedMessage id="workflowStates.title" defaultMessage="Workflow States" />}
```
Comment (rart) [inline ui/app/src/components/ItemStatesManagement/ItemStatesManagement.tsx:271]: Missing removing the duplicate translation (siteTools.itemStates) from json files

**Review by rart — CHANGES_REQUESTED** [review]

File: ui/app/src/components/ItemStatesManagement/ItemStatesManagement.tsx (hunk 4 above)
Comment (jvega190) [reply ui/app/src/components/ItemStatesManagement/ItemStatesManagement.tsx:271]: I did remove the translation, I'll comment on the lines where I removed it

File: ui/app/src/translations/locales/ko.json [hunk 5]
```diff
@@ -1375,7 +1375,6 @@
   "siteSearch.noOptionsAvailable": "사용 가능한 옵션이 없습니다",
   "siteSearch.unknownErrorSearching": "검색 서비스에 오류가 발생했습니다.",
   "siteSwitcherSelected.siteSelectorNoSiteSelected": "프로젝트 선택",
-  "siteTools.itemStates": "아이템 상태",
```
Comment (jvega190) [inline ui/app/src/translations/locales/ko.json:1378]: @rart ^

File: ui/app/src/translations/locales/es.json [hunk 6]
```diff
@@ -1375,7 +1375,6 @@
   "siteSearch.noOptionsAvailable": "No hay opciones disponibles.",
   "siteSearch.unknownErrorSearching": "Ocurrió un error con el servicio de búsqueda.",
   "siteSwitcherSelected.siteSelectorNoSiteSelected": "Elija proyecto",
-  "siteTools.itemStates": "Estados del artículo",
```
Comment (jvega190) [inline ui/app/src/translations/locales/es.json:1378]: @rart ^

File: ui/app/src/translations/locales/en.json [hunk 7]
```diff
@@ -1375,7 +1375,6 @@
   "siteSearch.noOptionsAvailable": "No options available",
   "siteSearch.unknownErrorSearching": "An error occurred with the search service.",
   "siteSwitcherSelected.siteSelectorNoSiteSelected": "Choose site",
-  "siteTools.itemStates": "Item States",
```
Comment (jvega190) [inline ui/app/src/translations/locales/en.json:1378]: @rart ^

File: ui/app/src/translations/locales/de.json [hunk 8]
```diff
@@ -1375,7 +1375,6 @@
   "siteSearch.noOptionsAvailable": "Keine Optionen verfügbar",
   "siteSearch.unknownErrorSearching": "Beim Suchdienst ist ein Fehler aufgetreten.",
   "siteSwitcherSelected.siteSelectorNoSiteSelected": "Projekt wählen",
-  "siteTools.itemStates": "Artikelstatus",
```
Comment (jvega190) [inline ui/app/src/translations/locales/de.json:1378]: @rart ^

**Review by rart — APPROVED** [review]

**Pull request merged**